src/main/java/id/co/veritrans/mdk/v1/gateway/model/vtweb/CreditCardOptions.java

```diff
@@ -0,0 +1,35 @@
+package id.co.veritrans.mdk.v1.gateway.model.vtweb;
+
+import id.co.veritrans.mdk.v1.gateway.model.vtdirect.paymentmethod.CreditCard;
+
+/**
+ * Set the credit card option such as the channel (eg. MIGS).
+ * Created by adampahlevi on 12/29/16.
```

Comment:
Not sure about convention in this project. But its recommended to turn author signature off in Intellij/Eclipse

Your contribution is already carved in history via git :)